docs/3.0.0-beta.x/cli/CLI.md

```diff
@@ -78,6 +78,63 @@ options: [--no-optimization]
 - **strapi build --no-optimization**<br/>
   Builds the administration panel without minimizing the assets. The build duration is faster.
 
+## strapi configuration:dump|config:dump
+
+Dumps configurations to a file or stdout to help you migrate to production.
+
+The dump format will be a JSON array.
+
+```
+strapi configuration:dump
+
+Options:
+  -f, --file <file>  Output file, default output is stdout
+```
+
+**Examples**
+
+- `strapi config:dump -f dump.json`
+- `strapi config:dump --file dump.json`
+- `strapi config:dump > dump.json`
+
+All these examples are equivalent.
+
+::: warning
+When configuring your application you often enter credentials for thrid party services (e.g authentication providers), Be aware that those credentials will also be dumped into the output of this command.
+When you aren't sure of what you are doing, you should avoid commiting this into a versioning system and rather copy the dump to the environment you want to restore it into when needed.
```

Comment:
```diff
- commiting
+ committing
```

Reply:
I had some difficulties to understand the end of the sentence. Would it be ok to add a comma ? `you want, to restore`

Reply:
With a comma the sentence doesn't have the same meaning so I can't :) Will rephrase to make it simpler 👍